ontobio/model/association.py

```diff
@@ -0,0 +1,63 @@
+import json
+import typing
+import collections
+import enum
+import datetime
+
+from typing import List, Optional, NamedTuple, Dict
```

Comment:
Added this `Dict` to typing. Was getting an error at the `Association.properties` declaration.